tensorflow/tools/dist_test/python/mnist_replica.py

```diff
@@ -88,34 +94,28 @@
 WORKER_PREFIX = "tf-worker"  # Prefix of the workers' domain names
 
 
-def get_device_setter(num_parameter_servers, num_workers):
-  """Get a device setter given number of servers in the cluster.
-
-  Given the numbers of parameter servers and workers, construct a device
-  setter object using ClusterSpec.
-
-  Args:
-    num_parameter_servers: Number of parameter servers
-    num_workers: Number of workers
+def get_cluster_setter():
```

Comment:
This function is confusingly named (what is a "cluster setter"? why does it return a server and a cluster?). Since it's only called from one place, I'd inline it at the call site, and the code will be easier to follow.